build.sh

```diff
@@ -164,6 +170,26 @@ if [ "${B_FS}" = "y" ] && [ -d  "${PREFIX_ROOTSKEL}" ]; then
 	install -m 664 "${PREFIX_BUILD}/git-version" "$PREFIX_FS/root/etc"
 fi
 
+#
+# Build host tools
+#
+if [ "${B_HOST}" = "y" ]; then
+	if [ "$TARGET" != "$_TARGET_FOR_HOST_BUILD" ]; then
+		# if not already building for host - re-exec with clean env
+		(env "$ORIG_ENV" TARGET=$_TARGET_FOR_HOST_BUILD ./phoenix-rtos-build/build.sh host)
+	else
+		source ./phoenix-rtos-build/build-host-tools.sh
+	fi
+fi
+
+# always install hostutils if they are present
+b_log "Installing hostutils"
+HOSTUTILS=(metaelf phoenixd psdisk psu syspagen)
+for tool in "${HOSTUTILS[@]}"; do
+	toolfile="$PREFIX_BUILD_HOST/prog.stripped/$tool"
+	[ -e "$toolfile" ] && cp -a "$toolfile" "$PREFIX_BOOT"
```

Comment:
I would probably use "test" command for more elegant syntax but it's rather personal preference

Reply:
common way in other scripts, so I followed. Didn't want to make full if-then-fi to make the code compact (not-so-important part of build.sh).
![image](https://github.com/phoenix-rtos/phoenix-rtos-build/assets/3939983/4d7da239-87c8-4fad-807d-bf263a9b6762)